content/Hardware Support/Shields and Carriers/How-to-calibrate-the-internal-temperature-sensor.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "How to calibrate the internal temperature sensor"
+id: 4411202645778
+---
+
+When working with internal temperature sensors, such as those found on boards like the Arduino MKR IoT Carrier or the Nicla Sense ME, one common issue is inaccurate temperature readings due to self-heating. Self-heating occurs because components on the board generate heat, which can lead to temperature readings higher than the actual environmental temperature. Factors like power usage, board positioning, and the running code can increase the impact of this effect.
```

Comment:
```suggestion
When using devices with internal temperature sensors, such as the Arduino Nicla Sense ME or the Arduino MKR IoT Carrier, a common issue is inaccurate temperature readings due to self-heating. This happens because heat generated by other components on the device can affect the sensor, leading to higher readings than the actual ambient temperature. Factors like power supply, board positioning, and the specific code running on the device can increase the impact of this effect.

Calibrating the internal sensor can correct these inaccuracies, giving you more accurate temperature readings.
```
Some tweaks to the intro to include all devices with internal sensors: boards and carriers.